docs/platform/concepts/service-power-cycle.rst

```diff
@@ -0,0 +1,39 @@
+Service power cycle
+===================
+
+Aiven service power off and power on is more than stopping and starting a service on nodes. For better utilisation of resources on Aiven platform, idle resources will be released and only the necessary data will be kept after power off. The impact on the service is different depending on the service type and plan. 
+
+.. warning:: Data loss may happen during a service power off, so it is important for users to understand the consequences before powering off a service.
+
+Aiven service power off and power on can be done on `Aiven Console <https://console.aiven.io>`_ or through :doc:`Aiven CLI </docs/platform/howto/pause-from-cli>`.
+
+Power off
+-------------
+
+After an Aiven service is powered off:
+
+* All virtual machine(s) of the service will be **removed** from the public cloud.
```

Comment:
Do we take a backup before removing the images from public cloud? Also, should we reference to them as cluster nodes?

Reply:
We backup service data and configurations etc. but not images. The images will be recreated during power up.

Virtual machine is used in many of our help documents (e.g. https://docs.aiven.io/docs/platform/concepts/availability-zones.html). It's commonly used to refer to instances(nodes) on clouds.